sw/device/boot_rom/bootstrap.c

```diff
@@ -112,7 +112,7 @@ static int bootstrap_flash(void) {
           LOG_INFO("Flash erase successful");
         }
 
-        if (flash_write(frame.header.flash_offset, frame.data,
+        if (flash_write(frame.header.flash_offset, DATA_PARTITION, frame.data,
```

Comment:
Where does `DATA_PARTITION` come from? Isn't it `kDataPartition`?

Reply:
ooops sorry i forgot to change that. I wonder why when i ran a ninja build i didn't see anything fail...

Reply:
updated, please take a look.